src/FixedDataTableCell.js

```diff
@@ -125,79 +126,76 @@ class FixedDataTableCell extends React.Component {
     return false;
   }
 
-  componentDidUpdate(prevProps) {
-    if (prevProps === this.props) {
-      return;
-    }
-
-    const props = this.props;
-    var left = props.left + this.state.displacement;
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
There's no net change in the function, I just reordered statements to make it cleaner.

Reply:
so is it just changing componentDidUpdate to getDerivedStateFromProps + reordering?

Reply:
Yep. This ensures a single render after state update in response to prop change.

Reply:
cool.  But to ensure I understand, we still get 2 rerenders, one for prop changes to the top level component, and another after the redux action is done?